tools/dpdk_ptf.Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM ghcr.io/ipdk-io/ipdk-ubuntu2004-x86_64:ipdk_v23.07
+
+# Prep
+ENV IPDK_INSTALL_DIR=/root/ipdk_install
+ENV PATH="${PATH}:/root/.local/bin"
+WORKDIR /root
+RUN rm -rf p4*
+COPY ./p4c /root/p4c
+COPY ./p4sde /root/p4sde
+COPY ./.ccache /root/.ccache
+RUN apt-get update
+RUN apt-get install rsync git ccache libhugetlbfs-bin -y
+RUN ccache -s
+
+# Update GCC to 11 -> GCC-9 cause config problem, remove after base OS is updated to 22.04
```

Comment:
I am guessing you are asking whether someone at Intel might publish a docker image with IPDK software already compiled and installed, rather than following instructions like these?  https://github.com/ipdk-io/ipdk/blob/main/build/networking/README_DOCKER.md

If so, let me know and I can try pinging a couple of people to find out if they have any intention of supporting such a thing.

Reply:
The docker image is built using a base OS that is Ubuntu 22.04, but the docker image itself is 20.04. That is normally fine, but the [p4-dpdk-target](https://github.com/p4lang/p4-dpdk-target) itself is built with Ubuntu 22.04. To get it to work on Ubuntu 20.04 requires a bunch of workarounds. It would be nice to remove this confounding factor. 

We thought we fixed the nondeterministic failures but that is sadly still not the case.